src/Annotation/Preferred.php

```diff
@@ -13,6 +13,7 @@
  * @Annotation
  * @Target({"METHOD", "PROPERTY"})
  */
+#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_PROPERTY)]
```

Comment:
Currently the whole project still supports annotations. Given that the library itself only supports PHP `> 8.0` and attributes are out for over 3 years already, should we drop annotation support, or just keep it, as it does not really hurt?